.github/ISSUE_TEMPLATE/server-migration.md

```diff
@@ -17,9 +19,10 @@ Checklist based on general guide https://github.com/openfoodfoundation/ofn-insta
 - [ ] DNS: add temporary domain (eg `prod2.openfoodnetwork.org`)
 
 ### config
-- [ ] Add temporary name to `inventory/hosts`
+- [ ] Add temporary name to `inventory/hosts` (suggest doing this on separate branch)
```

Comment:
I'm wondering about this _temporary_ host name. I don't think it's temporary. Don't we want to be able to identify a unique host like prod4.openfoodnetwork.org.au which is providing the site openfoodnetwork.org.au?